lib/common-types.ts

```diff
@@ -701,7 +701,7 @@ export type MRActions = {
     create: {
       params: GetAppActionCallParams
       payload: CreateAppActionCallProps
-      return: AppActionCallProps
+      return: WebhookCallDetailsProps
```

Comment:
`getCallDetails` function was using this type so I thought we can also return the same as are polling the api end point used in this function only.
https://github.com/contentful/contentful-management.js/blob/master/lib/adapters/REST/endpoints/app-action-call.ts#L20

Reply:
Lets rename it at least. Otherwise we'll be confusing end users - the two entities aren't really related and seems like an internal problem being exposed

Reply:
I realised this while writing test cases this function is not void, it returns the Promise<AppActionCallProps>  and this function is used like this in `app-action-utils`

```
const createResponse = await api.appActionCall.create(params, { parameters });
  // @ts-ignore
  const callId = createResponse.sys.id;
 ```
  
if we change the response like we did in the PR, they will now get `WebhookCallDetailsProps.sys.id`
so I am just worried if the dev-workflows team will be impacted by this change 🤔